cypress/cypress-smoketests/integration/user-post-authentication.cy.ts

```diff
@@ -1,63 +1,93 @@
-import { AuthenticationResult } from '../../support/login-types';
-import {
-  editPetitionerEmail,
-  goToCaseDetail,
-  verifyEmailChange,
-} from '../support/pages/case-detail';
-import { fillInCreateCaseFromPaperForm } from '../../cypress-integration/support/pages/create-paper-petition';
-import { getEnvironmentSpecificFunctions } from '../support/pages/environment-specific-factory';
-import { goToCaseDetailPetitioner } from '../support/pages/petitioner-dashboard';
-import {
-  goToCreateCase,
-  goToReviewCase,
-  serveCaseToIrs,
-} from '../support/pages/create-paper-case';
-import { goToMyDocumentQC } from '../support/pages/document-qc';
 import { v4 as uuidv4 } from 'uuid';
-const { closeScannerSetupDialogIfExists, login } =
-  getEnvironmentSpecificFunctions();
 
-const DEFAULT_ACCOUNT_PASS = Cypress.env('DEFAULT_ACCOUNT_PASS');
 const randomizedEmail = `${uuidv4()}@example.com`;
 
 if (!Cypress.env('SMOKETESTS_LOCAL') && !Cypress.env('MIGRATE')) {
-  describe('Petitions clerk', () => {
-    it('should be able to create a case with paper service', () => {
-      cy.task<AuthenticationResult>('getUserToken', {
-        email: 'petitionsclerk1@example.com',
-        password: DEFAULT_ACCOUNT_PASS,
-      }).then(result => {
-        login(result.AuthenticationResult?.IdToken);
+  describe('add email to practitioner', () => {
+    it('a noce should be generated after adding an email to a practitioner', () => {
+      cy.login('petitionsclerk1');
+      cy.get('table').should('exist');
+      cy.get(':nth-child(2) > .usa-nav__link').click();
+      cy.intercept('GET', 'https://**/dynamsoft.webtwain.initiate.js', {
+        body: `window.Dynamsoft = {DWT: {
+            GetWebTwain() {}
+          }}`,
+        statusCode: 200,
       });
```

Comment:
I love that you've figured this out. However, it doesn't seem to be the only place this would happen. Should this be put into a reusable helper?